brutils/phone.py

```diff
@@ -74,31 +74,64 @@ def remove_symbols_phone(phone_number):  # type: (str) -> str
     return cleaned_phone
 
 
-def _generate_ddd_number():  # type() -> str
-    """
-    Generate a valid DDD number.
+def generate(type=""):
```

Comment:
Trocando `""` por `None` para ficar igual is_valid_license_plate. Para manter o padrão. Também adicionei a notação dos argumentos e retorno

```suggestion
def generate(type=None): # type: (str) -> str
```